### PR TITLE
Add permalinks to headers (#189)

### PR DESCRIPTION
**Pull request by sgrossberndt**

### CODE REVIEW

**Comment by skinkie** [comment]
Is there a way to actually view this file in a browser?

**Comment by Aurige** [comment]
> Is there a way to actually view this file in a browser?

https://htmlpreview.github.io/?https://github.com/VDVde/OJP/blob/06b6b8abd44be234240a41d32cbf5cf1da2a36f0/docs/generated/OJP.html

**Comment by sgrossberndt** [comment]
That preview does not work properly, it seems to contain a JavaScript error. Easiest is to download OJP.html and asciidoc.css from https://github.com/VDVde/OJP/tree/06b6b8abd44be234240a41d32cbf5cf1da2a36f0/docs/generated and open OJP.html in a browser.

**Review by ue71603 — APPROVED** [review]

**Review by herlitze — APPROVED** [review]

File: docs/generated/OJP.html [hunk 1]
```diff
@@ -1,12 +1,54 @@
...
     <meta charset="UTF-8"/>
-    <meta http-equiv="Content-type" content="text/html;charset=UTF-8"/>
-    <meta name="robots" content="noindex nofollow"/>
+    <meta http-equiv="Content-Type" content="text/html;charset=UTF-8"/>
     <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <script type="text/javascript">
+					
+					document.addEventListener('DOMContentLoaded', function() {
+						var headers = ['h2', 'h3', 'h4', 'h5', 'h6'];
+						for (var i = 0; i &lt; headers.length; i++) {
```
Comment (skinkie) [inline docs/generated/OJP.html:11]: Is &lt; here how javascript works or an error when entering?

Comment (skinkie) [inline docs/generated/OJP.html:11]:
```suggestion
						for (var i = 0; i < headers.length; i++) {
```

File: docs/generated/OJP.html [hunk 2]
```diff
@@ -1,12 +1,54 @@
...
+							var headerElements = document.getElementsByTagName(headers[i]);
+							for (var j = 0; j &lt; headerElements.length; j++) {
```
Comment (skinkie) [inline docs/generated/OJP.html:13]:
```suggestion
							for (var j = 0; j < headerElements.length; j++) {
```

File: docs/generated/OJP.html [hunk 3]
```diff
@@ -1,12 +1,54 @@
...
+								var header = headerElements[j];
+								header.innerHTML += '&lt;a class="header-link" href="#' + header.parentNode.id + '"&gt;&lt;span class="link-icon"&gt;&lt;/span&gt;&lt;/a&gt;';
```
Comment (skinkie) [inline docs/generated/OJP.html:15]:
```suggestion
								header.innerHTML += '<a class="header-link" href="#' + header.parentNode.id + '"><span class="link-icon"></span></a>';
```

**Review by herlitze — APPROVED** [review]

**Review by ue71603 — APPROVED** [review]

**Pull request merged**